File: CompilationTest/MapTest/results/MapTest.ts
class TestClass {
    getResult() {
        const mapObj = {
            x: 5
        };
        //let containsX = "x" in mapObj;
        //delete mapObj["x"];
        mapObj["x"] = 3;
        return mapObj["x"];
    }
  
    testMethod() {
        console.log(`Result = ${this.getResult()}`);
    }
}

try {
    new TestClass().testMethod();
} catch(e) {
    console.log(`Exception: ${e.message}`);
}

File: CompilationTest/MapTest/results/MapTest.py
class TestClass:
    def get_result(self):
        map_obj = {
          "x": 5,
        }
        #let containsX = "x" in mapObj;
        #delete mapObj["x"];
        map_obj["x"] = 3
        return map_obj["x"]
    
    def test_method(self):
        print "Result = %s" % (self.get_result())

try:
    TestClass().test_method()
except Exception as err:
    print "Exception: " + err.message